app.py

```diff
@@ -1,11 +1,65 @@
-from flask import Flask
+from flask import Flask, render_template, url_for, redirect
+from flask_sqlalchemy import SQLAlchemy
+from flask_wtf import FlaskForm
+from wtforms import StringField, SubmitField
+from wtforms.validators import DataRequired
```

Comment:
Since you are directly using wtforms, you should include it in your `requirements.txt`.

Reply:
Fixed in next commit